https/README.md

```diff
@@ -25,7 +25,7 @@ tls_config:
 
 # List of usernames and hashed passwords that have full access to the web
 # server via basic authentication. If empty, no basic authentication is
-required. Passwords are hashed # with bcrypt.
+# required. Passwords are hashed with bcrypt.
 users:
```

Comment:
`users` doesn't really say basic auth to me, maybe `basic_auth_users`?

Reply:
good idea